public/data/devmukhtarr.json

```diff
@@ -24,5 +24,23 @@
       "url": "https://www.linkedin.com/in/abdulfatai-mukhtar-77a2131b1/",
       "icon": "linkedIn"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Wrote my first blog post on hashnode",
+      "date": "Feb 6, 2021",
+      "icon": "hashnode",
+      "color": "blue",
+      "description": "Wrote my first ever blog post which was about setting up nodejs and published it on hashnode",
+      "url": "https://mukhtar.hashnode.dev/setting-up-nodejs"
+    },
+    {
+      "title": "Completed Udacity Fullstack Nanodegree sponsored by ALX-T",
+      "date": "Aug 19, 2022",
+      "icon": "book",
+      "color": "blue",
+      "description": "I completed and learnt more about fullstack development with Udacity course",
+      "url": "https://graduation.udacity.com/confirm/GFVHADAL"
```

Comment:
**Not sure what this url is about, is it the certificate of completion!** 
It's taking me directly to the sign in page at udacity.

I recommend using a public url, that everyone can see or have access to, or may be the course link!

Reply:
It is the the link to the certificate which is made public

Reply:
Still it's redirecting me to the sign in page, not sure why!!

Reply:
Can there be milestone alone without link so I remove the certificate link

Reply:
Yes, you can remove the url key and it will still work.

Reply:
I have removed the URL